drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -843,8 +843,9 @@ int bnxt_hwrm_vnic_cfg(struct bnxt *bp, struct bnxt_vnic_info *vnic)
 	HWRM_PREP(req, VNIC_CFG, -1, resp);
 
 	/* Only RSS support for now TBD: COS & LB */
-	req.enables =
-	    rte_cpu_to_le_32(HWRM_VNIC_CFG_INPUT_ENABLES_DFLT_RING_GRP |
+	if (!vnic->bd_stall)
```

Comment:
This breaks the function if bd_stall is true.  I don't see a reason for these enables bits not to be set when bd_stall is true.